test_run.sh

```diff
@@ -525,6 +525,23 @@ revert_change() {
   mv $1/$2.bak $1/$2
 }
 
+test_intellij_aspect() {
+  intellij_git_tag=$1
+  rules_scala_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+  if [[ "$runner" == "run_test_ci" ]]; then
+    # ci: intellij is checked out with full git history in `.travis.yaml`
+    cd intellij
+  else
+    # local: update or checkout a sibling dir.
+    cd "${rules_scala_dir}/../"
+    test -d "intellij/.git" || git clone git@github.com:bazelbuild/intellij.git
+    cd intellij && git fetch && git pull
+  fi
+  git checkout "${intellij_git_tag}"
+  bazel test --test_output=errors --incompatible_disallow_set_constructor=false --override_repository io_bazel_rules_scala=${rules_scala_dir} //aspect/testing/tests/src/com/google/idea/blaze/aspect/scala/...
```

Comment:
you should quote `${rules_scala_dir}` in case it has whitespace

Reply:
Good idea, I'll update the PR.